### PR TITLE
Prepare release notes for 1.0.5

### PR DESCRIPTION
**Pull request by miekg**

### CODE REVIEW

File: content/blog/coredns-1.0.5.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+* *log* adds message ID `{>id}` to the default logging.
+* *health* has gotten a lameduck option that will nack health, but will keep the server running for a configurable duration when CoreDNS is being shut down. If metrics are enabled *health* exports a metric that curls the local endpoint and exports the duration. Useful for getting a sense of overloadedness of the process.
+* *rewrite* can now rewrite answers for `name regex` matches. This prevents DNS clients from ignoring the answers due to a mismatch with the original question.
+
+## Contributors
+
+The following people helped with getting this release done:
+
+Christian Nilsson,
+cricketliu,
```
Comment (fturib) [inline content/blog/coredns-1.0.5.md]: name is : Cricket Liu

**Comment by miekg** [comment]
[ Quoting <notifications@github.com> in "Re: [coredns/coredns.io] Prepare re..." ]
>fturib commented on this pull request.
>
>
>
>> +
>+* A new plugin *route53* was added that enables serving zone data from AWS route53, see the [documentation](https://coredns.io/plugins/route53).
>+* A new plugin *on* was added. This is an external Caddy [plugin](https://caddyserver.com/docs/on), that is now also available (by default) for CoreDNS; it allows you to run commands when an event is generated.
>+
>+* *cache* doesn't apply a 5s minimum TTL anymore. It fixes prefetching *and* correctly sets the metrics for cache hits and misses.
>+* *dnssec* fixes handing out *expired* signatures after 8 days and properly filters out the qtype in the NSEC bitmap for NXDOMAIN responses.
>+* *log* adds message ID `{>id}` to the default logging.
>+* *health* has gotten a lameduck option that will nack health, but will keep the server running for a configurable duration when CoreDNS is being shut down. If metrics are enabled *health* exports a metric that curls the local endpoint and exports the duration. Useful for getting a sense of overloadedness of the process.
>+* *rewrite* can now rewrite answers for `name regex` matches. This prevents DNS clients from ignoring the answers due to a mismatch with the original question.
>+
>+## Contributors
>+
>+The following people helped with getting this release done:
>+
>+Christian Nilsson,
>+cricketliu,
>
>name is : Cricket Liu

true, but this is an author dump from git - so it is what it is

**Comment by miekg** [comment]
ping @cricketliu you may want to properly set your .gitconfig's author section?

**Comment by cricketliu** [comment]
Will do!

Cricket

On Jan 25, 2018, at 8:38 AM, Miek Gieben <notifications@github.com<mailto:notifications@github.com>> wrote:


ping @cricketliu<https://github.com/cricketliu> you may want to properly set your .gitconfig's author section?

**Pull request merged**